src/lib/import-export/import/handlers/backup-handler-zip.ts

```diff
@@ -47,6 +48,17 @@ export class BackupHandlerZip extends EventEmitter implements BackupHandler {
 		this.emit( ImportEvents.BACKUP_EXTRACT_START );
 
 		return new Promise( ( resolve, reject ) => {
+			let extractionFailed = false;
+			function failOnce( err: Error, context?: Record< string, unknown > ) {
```

Comment:
```suggestion
			const failOnce = (err: Error, context?: Record<string, unknown>) => {
```
Can we use an inline function instead of an arrow function for `failOnce`?